docs/en/glossary/glossary.asciidoc

```diff
@@ -373,6 +373,15 @@ order to index <<glossary-event,event>> data.
 +
 //Source: Logstash
 endif::logstash-terms[]
+ifdef::xpack-terms[]
+[[glossary-influencer]] influencer ::
+
+Influencers are fields that affect whether a bucket is anomalous in an
```

Comment:
Hi Lisa!
I've checked how we talked about influencers in some past blog posts and I think it would be better to revise this glossary entry to be "Influencers are entities that may have contributed to an anomaly in a given bucket." to be more in line with our previous writing on this topic. 
Do you think we should mention things like "These entities will, in practice, be values of fields in the documents of the analyzed dataset"? Or is this too much for a glossary?

Reply:
Thanks @Winterflower I've changed it to this:
> Influencers are entities that might have contributed to an anomaly in a specific
bucket in an {anomaly-job}.

Reply:
Thank you @lcawl - it looks good!